### PR TITLE
Work around for live reload with non 127.0.0.1 IP

### PR DESCRIPTION
**Pull request by cameron-elliott**
Actual Issue link: https://github.com/facebook/docusaurus/issues/1753

### CODE REVIEW

File: packages/docusaurus-1.x/lib/server/liveReloadServer.js [hunk 1]
```diff
@@ -27,7 +27,21 @@ function start(port) {
     },
   );
 }
+
 const getReloadScriptUrl = () => {
+  // Please, JS programmer show me how to save state of 'program' globally for each
```
Comment (yangshun) [inline packages/docusaurus-1.x/lib/server/liveReloadServer.js]: Use `process.env.LIVERELOAD_HOST` instead of hacking into `commander`?

Comment (cameron-elliott) [reply packages/docusaurus-1.x/lib/server/liveReloadServer.js]: I have decided to go another way for live reload, so I am withdrawing this PR.

**Comment by cameron-elliott** [comment]
Withdrawing PR

**Comment by yangshun** [comment]
Sure thing, thanks for the update!

**Pull request closed without merging**